sys/net/sock/sock_util.c

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
Would prefer this file in directory `sys/net/sock/util`, in case there will be general binaries for `sock` one day.